tracer/build/_build/Honeypot/IntegrationGroups.cs

```diff
@@ -187,8 +187,8 @@ private async Task PopulatePackages(List<PackageVersionGenerator.TestedPackage>
 
                 if (latestSupportedPackage is null)
                 {
-                    Logger.Warning($"No version of {packageName} below maximum package version {MaximumSupportedAssemblyVersion}." +
-                                $"Using latest instead");
+                    Logger.Warning($"No version of {packageName} below maximum package version {MaximumSupportedAssemblyVersion}. " +
```

Comment:
I don't really understand _exactly_ what these _mean_

For example the `OpenTelemetry` instrumentations hit this: https://github.com/DataDog/dd-trace-dotnet/blob/45a66abc870d1d45ce0778909162203e55dcfa2a/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/OpenTelemetry/TracerProviderBuilderIntegration.cs#L31

Reply:
Theoretically, this means that we only support (for example) 1.x.x-2.x.x, but the only package it can find is 3.x.x...

Generally it means the mapping between assemblies and nugets is a mess and we can't always infer stuff as we'd like